framework/src/org/checkerframework/common/signatureinference/SignatureInferenceScenes.java

```diff
@@ -0,0 +1,901 @@
+package org.checkerframework.common.signatureinference;
```

Comment:
I split this into 3 classes:
`WholeProgramInferenceScenes`
`WholeProgramInferenceScenesHelper`
`AnnotationConverter`